### PR TITLE
[ticket/13101] Remove MSN/WLM custom profile field

### PR DESCRIPTION
**Pull request by Zoddo**
[PHPBB3-13101](https://tracker.phpbb.com/browse/PHPBB3-13101)

### CODE REVIEW

**Comment by Zoddo** [comment]
Rebased to fix conflicts (old commit: 94ec052)

File: phpBB/phpbb/db/migration/data/v320/remove_profilefield_wlm.php [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+					'pf_phpbb_wlm'		=> array('VCHAR', ''),
+				),
+			),
+		);
+	}
+
+	public function update_data()
+	{
+		return array(
+		array('custom', array(array($this, 'delete_custom_profile_field_data'))),
```
Comment (marc1706) [inline phpBB/phpbb/db/migration/data/v320/remove_profilefield_wlm.php]: Indentation is off on this line and the next one.

Comment (Zoddo) [inline phpBB/phpbb/db/migration/data/v320/remove_profilefield_wlm.php]: @marc1706 : Fixed :)

**Comment by marc1706** [comment]
:+1:

**Pull request merged**